lib/Textile/API/ipfs.ts

```diff
@@ -2,6 +2,11 @@ import { NativeModules } from 'react-native'
 
 const { TextileNode } = NativeModules
 
+export async function dataAtPath(path: string): Promise<string> {
+  const result = await TextileNode.dataAtPath(path)
+  return result as string
```

Comment:
`Buffer` (`UIntArray`) could be a better return type for "data".

Reply:
Or is the string returned a data url?

Reply:
The data could be anything at that path. So, no way to properly construct a data URL. It just returns a byte slice: `func (m *Mobile) DataAtPath(pth string) ([]byte, error)`

Reply:
Makes sense. This should return `Buffer` or `UIntArray` then.

Reply:
ah okay, i just got lucky in that the string could just be dropped into the rn image uri

Reply:
Surprised that worked. Maybe RN `Image` has some default handling of string values that made it work.